components/StyledSelect.js

```diff
@@ -5,7 +5,7 @@ import { typography, layout, space } from 'styled-system';
 import Select, { components } from 'react-select';
 import { injectIntl, defineMessages } from 'react-intl';
 import propTypes from '@styled-system/prop-types';
-import { truncate } from 'lodash';
+import { truncate, unescape } from 'lodash';
```

Comment:
I'm starting to think I should unescape this in the backend, will update this tomorrow.